### PR TITLE
Story/em 98 filter metrics by period

### PR DESCRIPTION
**Pull request by hvilloria**
## What does this PR do?

* changes the select filter (weekly/daily) by a number input to return the weeks with metrics
* removed unused classes
* all the daily proccess classes were not removed in order to used later

interval-filter

### CODE REVIEW

**Comment by hvilloria** [comment]
`app/services/code_climate_summary_retriever.rb` is failing for code climate analisys, when i start working on Martin's PR i will do the fix

**Comment by hdamico** [comment]
Can you attach a gif showing the new feature?

File: app/assets/stylesheets/metrics/_interval-filter.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+}
+
+.period-selection{
+  label.numeric{
+    width: 13rem;
+  }
+  input.numeric{
+    margin: 0 0.2rem;
+  }
+}
```
Comment (hdamico) [inline app/assets/stylesheets/metrics/_interval-filter.scss:12]: Is this fully responsive? The 13 rem looks weird.

Comment (hvilloria) [reply app/assets/stylesheets/metrics/_interval-filter.scss:12]: yep, it is just a with to separate the input from the label

**Pull request merged**